src/pages/concept-registration-page/form-concept/form-concept.component.tsx

```diff
@@ -63,7 +65,9 @@ const config = {
       fagområde = '',
       bruksområde = [],
       omfang = null,
-      kontaktpunkt = null
+      kontaktpunkt = null,
+      gyldigFom = '',
+      gyldigTom = ''
```

Comment:
Disse to trenger ikke noe fallback verdier, siden de kan være enten string eller undefined

Reply:
Rettet